packaging/centos/datadog-agent.init

```diff
@@ -31,7 +31,7 @@ FORWARDERPATH="/opt/datadog-agent/agent/ddagent.py"
 SUPERVISORD_PATH="PATH=/opt/datadog-agent/embedded/bin:/opt/datadog-agent/bin:$PATH /opt/datadog-agent/bin/supervisord"
 SUPERVISORCTL_PATH="/opt/datadog-agent/bin/supervisorctl"
 SUPERVISOR_CONF="/etc/dd-agent/supervisor.conf"
-SUPERVISOR_SOCK="/var/tmp/datadog-supervisor.sock"
+SUPERVISOR_SOCK="/opt/datadog-agent/run/datadog-supervisor.sock"
```

Comment:
It shouldn't be an issue, but we'll need to triple check that it doesn't cause any trouble while upgrading.